### ISSUE TITLE
Installing DaCy downgrades huggingface-hub to 0.0.12, leading to incompatible load_dataset

### ISSUE THREAD

**Issue by MartinBernstorff** [centre-for-humanities-computing/DaCy#93]
## How to reproduce the behaviour
1. Install daCy
2. `from datasets import load_dataset`

Gives me an error :-)

**Comment by KennethEnevoldsen** [comment]
@martbern can you provide details of your environment. If you're using spaCy 1.7+, you can type `python -m spacy info --markdown` and copy-paste the result here along with the DaCy version which you can get using `python -c "import dacy; print(dacy.__version__)"`.-->

**Comment by martbern** [comment]
Sure thing!

```
## Info about spaCy

- **spaCy version:** 3.2.1
- **Platform:** macOS-12.0.1-arm64-arm-64bit
- **Python version:** 3.8.10

python -c "import dacy; print(dacy.__version__)"
1.3.0
```

**Comment by KennethEnevoldsen** [comment]
This will probably be solved when #90 is merged
